src/fights.ts

```diff
@@ -361,20 +358,28 @@ function pygmyOptions(forceEquip: Item[] = []) {
 let _bestShadowRift: Location | null = null;
 export function bestShadowRift(): Location {
   if (!_bestShadowRift) {
-    _bestShadowRift =
-      chooseRift({
-        canAdventure: true,
-        sortBy: (l: Location) => {
-          const drops = getMonsters(l)
-            .map((m) => Object.keys(itemDrops(m)).map((s) => toItem(s)))
-            .flat();
-          return sum(drops, garboValue);
-        },
-      }) ?? $location.none;
+    _bestShadowRift = ClosedCircuitPayphone.chooseRift({
+      canAdventure: true,
+      sortBy: (l: Location) => {
+        setLocation(l);
+        // We probably aren't capping item drops with the penalty
+        // so we don't really need to compute the actual outfit (or the dropModifier for that matter actually)
+        const dropModifier = 1 + numericModifier("Item Drop") / 100;
+        return sum(getMonsters(l), (m) => {
+          const monsterDrops = itemDrops(m);
+          const items = Object.keys(monsterDrops).map(toItem);
```

Comment:
Just as a heads up, taking native mafia functions and dropping them directly in like that will typically cause errors due to differences in the ways js and ash handle overloads; mafia will say "there's no such thing as `to_item(string, number, string[]`"

Reply:
That will be moot due to the below comment, but it's still a good note